layouts/_default/baseof.html

```diff
@@ -4,17 +4,17 @@
 {{ partial "head.html" . }}
 
 <body>
-    {{ partial "header.html" . }}
+{{ partial "header.html" . }}
```

Comment:
Things like `{{ partial`, `{{ block`, and `{{ .Content`, where HTML is being pulled in from another file, have been moved to the beginning of the line. If the included file wants indentation it can do it on its own. This keeps it consistent. Otherwise the first line of the output HTML was indented and the rest of the lines were not indented and that looked weird.